template-html5-app/package.json

```diff
@@ -17,7 +17,7 @@
     "format:check": "prettier 'src/**/*.{js,jsx,ts,tsx,css,md,json}' --check --config ../.prettierrc.js --cache --cache-location=../prettiercache",
     "format": "prettier --write 'src/**/*.{js,jsx,ts,tsx,css,md,json}' --config ../.prettierrc.js && npm run lint",
     "test": "jest --config jest.config.js src/",
-    "publish": "npm publish --access public"
+    "publish-package": "npm publish --access public"
```

Comment:
Doesn't every example/template still need a "publish" script?

Reply:
no, because `publish` is a built-in npm command and when you name a script `publish` you get weird behavior https://stackoverflow.com/questions/71730319/npm-attempts-to-publish-twice